styles/markdown.scss

```diff
@@ -0,0 +1,39 @@
+// Default styles to apply to markdown generated content
+
+@import "variables.scss";
+
+.rendered-markdown {
+  margin-bottom: var(--spacing-xl);
+
+  h1 {
+    margin-top: var(--spacing-md);
+  }
+  ul {
+    list-style: disc;
+    padding-left: 2rem;
+    margin: var(--spacing-sm) 0;
+  }
+  ol {
+    list-style: georgian;
```

Comment:
This looks cool but I think they just use decimal haha

<img width="737" alt="Screenshot 2024-04-09 at 9 29 32 AM" src="https://github.com/bocoup/p5.js-website/assets/10382506/96e4f22d-b4f8-4a2f-873d-62b7efc561d5">

Reply:
loool 🤦🏼‍♀️